source/renderer/app/components/staking/delegation-setup-wizard/DelegationStepsActivationDialog.js

```diff
@@ -0,0 +1,263 @@
+  },
+  postponeButtonLabel: {
+    id: 'staking.delegationSetup.activation.step.dialog.postponeButtonLabel',
+    defaultMessage: '!!!I’ll do it later',
+    description:
+      'Postpone button label on the delegation setup "activation" step dialog.',
+  },
+});
+
+type State = {
+  spendingPasswordValue: string,
+};
+
+type Props = {
+  isSpendingPasswordSet?: boolean,
+  onActivate: Function,
+  onBack: Function,
+  onClose: Function,
+  stepsList: Array<string>,
+};
+
+export default class DelegationStepsActivationDialog extends Component<
+  Props,
+  State
+> {
+  static contextTypes = {
+    intl: intlShape.isRequired,
+  };
+
+  state = {
+    spendingPasswordValue: '',
+  };
+
+  form = new ReactToolboxMobxForm({
+    fields: {
+      spendingPassword: {
+        type: 'password',
+        label: this.context.intl.formatMessage(messages.spendingPasswordLabel),
+        placeholder: this.context.intl.formatMessage(
+          messages.spendingPasswordPlaceholder
+        ),
+        value: '',
+      },
+    },
+  });
+
+  submit = () => {
+    this.form.submit({
+      onSuccess: () => {
+        const { isSpendingPasswordSet } = this.props;
+        const data = {
+          fees: 12.042481,
+          amount: 3,
+          total: 15.042481,
+          password: isSpendingPasswordSet
+            ? this.state.spendingPasswordValue
+            : null,
+        };
+        this.props.onActivate(data);
+      },
+      onError: () => {},
+    });
+  };
+
+  handlePasswordChange = (value: string) => {
+    this.setState({ spendingPasswordValue: value });
+  };
+
+  render() {
+    const { form } = this;
+    const { intl } = this.context;
+    const { isSpendingPasswordSet, onBack, onClose, stepsList } = this.props;
+    const { spendingPasswordValue } = this.state;
+
+    const spendingPasswordField = form.$('spendingPassword');
+
+    const actions = [
+      {
+        className: 'postponeButton',
+        label: intl.formatMessage(messages.postponeButtonLabel),
+        onClick: onClose,
+      },
+      {
+        className: 'confirmButton',
+        label: intl.formatMessage(messages.confirmButtonLabel),
+        onClick: this.submit,
+        primary: true,
+      },
+    ];
+
+    return (
+      <Dialog
+        title={intl.formatMessage(messages.title)}
+        actions={actions}
+        closeOnOverlayClick
+        onClose={onClose}
+        className={styles.delegationStepsActivationDialogWrapper}
+        closeButton={<DialogCloseButton onClose={onClose} />}
+        backButton={<DialogBackButton onBack={onBack} />}
+      >
+        <div className={styles.delegationStepsIndicatorWrapper}>
+          <p className={styles.stepIndicatorLabel}>
+            <FormattedMessage
+              {...messages.stepIndicatorLabel}
+              values={{
+                currentStep: 4,
+                totalSteps: stepsList.length,
+              }}
+            />
+          </p>
+          <Stepper
```

Comment:
This can be done later

Reply:
@daniloprates I thought about that but as we are really tight with time maybe we can improve that in some new PR.

- Steps indicator label is actually Dialog subtitle since need to be fixed but also maybe we will have discussion later to make also `Stepper` fixed on top of dialog and then we can round whole story with one new component.
What you think?